### ISSUE TITLE
iOS Unity Build Fails Due to Conflicting Pods in AppsFlyer Ad Revenue Generic Connector and AppsFlyer Unity SDK

### ISSUE THREAD

**Issue by mryildirim91** [AppsFlyerSDK/appsflyer-unity-adrevenue-generic-connector#11]
Unity version: 2022.3.25f1
AppsFlyer SDK version: 6.14.4
AppsFlyer Ad Revenue Generic Connector version: 6.9.42

Platform: iOS

Description

We are attempting to integrate the AppsFlyer Ad Revenue Generic Connector with the AppsFlyer Unity SDK in a Unity project targeted for iOS. However, we are encountering issues with the Pod installation process. The Unity Editor fails to build, citing a failed pod install command.

Steps to Reproduce

Set up a Unity project with AppsFlyer Ad Revenue Generic Connector and AppsFlyer Unity SDK.
Attempt to build for iOS.
Observe the pod installation failure.

Error:
**_[!] The 'Pods-UnityFramework' target has frameworks with conflicting names: appsflyerlib.xcframework._**

Podfile:
[Podfile.txt](https://github.com/user-attachments/files/15902733/Podfile.txt)

Screenshots:

![Screen Shot 2024-06-19 at 17 46 06 PM](https://github.com/AppsFlyerSDK/appsflyer-unity-adrevenue-generic-connector/assets/61324574/73f89fa8-1da2-4ee0-8776-8ae05b6e4cc6)

It appears the conflict arises because the AppsFlyer Ad Revenue Generic Connector iOS Pod has a dependency on AppsFlyerFramework, and since our project also requires AppsFlyerFramework/Static, both dependencies are installed, leading to a conflict. We are looking for guidance on how to resolve this issue.

![Screen Shot 2024-06-19 at 17 58 46 PM](https://github.com/AppsFlyerSDK/appsflyer-unity-adrevenue-generic-connector/assets/61324574/8441e5a8-bb81-49e6-ab61-e545fbde2b4b)

**Comment by af-vero** [comment]
Hi, please pay attention that AppsFlyer Ad Revenue Generic Connector version  6.9.42 supports Unity AppsFlyer version 6.14.3 and not 6.14.4
We will update it soon, but for now please use 6.14.3
Thanks

**Comment by mryildirim91** [comment]
Actually we were using the 6.14.3 version and still getting the same error. We updated it in the hope of fixing the problem but nothing changed.

**Comment by af-vero** [comment]
Sorry for the confusion. Unity SDK should be 6.14.3 as the connector - the latest version 6.14.3
If you want to use older version of the connector, please use older version of the Unity SDK.

**Comment by mryildirim91** [comment]
Actually we were using Unity SDK 6.14.3 version in the first place. Yet we were still getting the same error. Downgrading the Unity Sdk version does not fix our problem because this problem first happened when we had Unity SDK version 6.14.3 and AppsFlyer Ad Revenue Generic Connector version 6.9.42

**Comment by af-vero** [comment]
ok, so if you use 6.14.3 Unity SDK, use 6.14.3 AdRevenue Connector as well

**Comment by mryildirim91** [comment]
When try to install it from the GitHub url that is from your documentation page, it automatically installs version 6.9.42. Then I changed the version in the git url like this  "https://github.com/AppsFlyerSDK/appsflyer-unity-adrevenue-generic-connector.git#v6.14.3-upm". It installs the version 6.14.3 but this time "AppsFlyerAdRevenue" class is not available. Lastly I tried installing version 6.14.3 importing the _**appsflyer-unity-adrevenue-generic-connector-6.14.3.unitypackage**_ you provided here: [](https://github.com/AppsFlyerSDK/appsflyer-unity-adrevenue-generic-connector). But I still have the same problem when I try to build on iOS.
